queries/odin/highlights.scm

```diff
@@ -151,6 +152,10 @@
   (identifier) @type
   "::")
 
+(bit_field_declaration
+  (identifier) @type
+  "::")
+
 (const_declaration
   (identifier) @type
```

Comment:
We're getting a little bit too much of this pattern that I want to ask, can this be possible
```suggestion
((identifier) @type
  .
  "::")
```

Reply:
I think that would match constants `foo :: 1` and highlight it as a type too

Reply:
makes sense then. Consider this resolved